vault/resource_database_secret_backend_connection_test.go

```diff
@@ -692,13 +702,15 @@ resource "vault_database_secret_backend_connection" "test" {
   }
 
   data = {
-	  password = "%s"
+	  password            = "%s"
+	  tls_ca              = "%s"
+	  tls_certificate_key = "%s"
```

Comment:
I think these need to be in the `mysql {}` block?

And I'm thinking they will also need to be multiline strings, something like:

```suggestion
	  tls_ca              = <<EOT
%s
EOT
	  tls_certificate_key = <<EOT
%s
EOT
```

Of course, that will mean they're not exactly empty strings when not set for the tests, so I'd still like to see the tls options in a new test, instead of adding them to the existing tests. i.e. add a new `testAccDatabaseSecretBackendConnectionConfig_mysql_tls()` function, used in a new `TestAccDatabaseSecretBackendConnection_mysql_tls()` test.